docs/contributing/coding-style.md

```diff
@@ -965,3 +965,109 @@ We do not suggest to modify the generated code other than the `tests` variable,
         type want struct {
             // generated test code
     ```
+### Mock
+
+There is a lot of dependency between each package. 
+
+As a result, the complexity of the test has increased, and it has become more difficult to determine whether or not to mock dependent objects.
```

Comment:
I think we should also mention the risk of using mock, and we should also mention to avoid using mock unless the test is extremely difficult to implement